bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Title.java

```diff
@@ -38,6 +38,13 @@
      */
     String PN_DESIGN_DEFAULT_TYPE = "type";
 
+    /**
+     * Name of the policy property that defines whether or not the title link is disabled.
+     *
+     * @since com.adobe.cq.wcm.core.components.models 12.5.0
+     */
+    String PN_TITLE_LINK_DISABLED = "linkDisabled";
```

Comment:
I would favor a positive property name `linkEnabled` instead.

Reply:
@jckautzmann - what's the benefit of the positive property? I thought we took the approach that the input is by default unchecked.

Reply:
@richardhand That's what I thought since the link option of Title is by default enabled. Similar is the case in Teaser component i.e. the `titleLinkHidden` property.

Reply:
I thought that we once had such a convention in AEM (use positive property names) but I can't find a reference now.

Reply:
I also prefer the situation where all defaults are unchecked. I'm fine if we keep the current approach.